__docs__/wonder-blocks-card/accessibility.mdx

```diff
@@ -33,7 +33,7 @@ The `inert` prop can be used to remove cards from the accessibility tree and key
 - Sets the `inert` attribute on the card container
 - All interactive elements within the card become non-focusable
 
-See the [In a Stack](./?path=/story/packages-card-card--in-a-stack) story for an example of disabling cards while keeping them part of a list count, like End-of-Task cards.
+See the In a Stack story for an example of disabling cards while keeping them part of a list count, like End-of-Task cards.
```

Comment:
These links are broken, and it was easier to just unlink them. I opened an issue recently for related work: https://khanacademy.atlassian.net/browse/WB-2108